proxy/src/compute.rs

```diff
@@ -219,6 +218,16 @@ impl ConnCfg {
 
         Ok(connection)
     }
+
+    /// Connect to a corresponding compute node.
+    async fn connect(&self) -> Result<PostgresConnection, ConnectionError> {
```

Comment:
Funny, vscode rust-analyzer breaks here; it think this is never called, though I see `proxy.rs:271` doing that. Instead it thinks this is tokio-postgres `Config` `connect`.

Reply:
This is a bug on my side :) Thanks, this should not be the case.

Reply:
Ah, so there is Deref...